### PR TITLE
Update Scala Version

### LINKED ISSUE #7376: Shall we add support for scala 2.13.9?
`CVE-2022-36944` exists in scala 2.13.0 ~ 2.13.8, see https://nvd.nist.gov/vuln/detail/CVE-2022-36944
Are we going to add support for scala 2.13.9?

### PR DESCRIPTION
**Pull request by adamgfraser**
Resolves #7376.

### CODE REVIEW

**Review by vigoo — DISMISSED** [review]

**Review by Wosin — DISMISSED** [review]

**Comment by narma** [comment]
Looks like we should wait for 2.13.10 - 
- https://github.com/scala/bug/issues/12650
- https://twitter.com/scala_lang/status/1575548918858887168

**Comment by sideeffffect** [comment]
The 2.13.10 release is [rolling out now](https://contributors.scala-lang.org/t/scala-2-13-10-release-planning/5943/9). AFAIK we just need to wait for Silencer ( https://repo1.maven.org/maven2/com/github/ghik/silencer-lib_2.13.10/ ) and then we can upgrade too :crossed_fingers: 
EDIT: relevant ticket https://github.com/ghik/silencer/issues/77

**Comment by sideeffffect** [comment]
@adamgfraser new Silencer for 2.13.10 has been released. We can now upgrade to 2.13.10, Silencer [1.7.12](https://repo1.maven.org/maven2/com/github/ghik/silencer-lib_2.13.10/).

Maybe we won't need all the changes to the code with 2.13.10 (unlike with 2.13.9).

File: project/plugins.sbt [hunk 1]
```diff
@@ -15,6 +15,6 @@ addSbtPlugin("org.scalameta"                     % "sbt-mdoc"
 addSbtPlugin("org.scalameta"                     % "sbt-scalafmt"                  % "2.4.6")
 addSbtPlugin("pl.project13.scala"                % "sbt-jcstress"                  % "0.2.0")
 addSbtPlugin("pl.project13.scala"                % "sbt-jmh"                       % "0.4.3")
-addSbtPlugin("ch.epfl.scala"                     % "sbt-scalafix"                  % "0.10.1")
+addSbtPlugin("ch.epfl.scala"                     % "sbt-scalafix"                  % "0.10.3")
```
Comment (sideeffffect) [inline project/plugins.sbt]: needs to be updated to 0.10.4

File: build.sbt [hunk 2]
```diff
@@ -764,7 +764,7 @@ lazy val scalafixRules = project.module
   .settings(
     scalafixSettings,
     semanticdbEnabled                      := true, // enable SemanticDB
-    libraryDependencies += "ch.epfl.scala" %% "scalafix-core" % "0.10.1"
+    libraryDependencies += "ch.epfl.scala" %% "scalafix-core" % "0.10.3"
```
Comment (sideeffffect) [inline build.sbt]: needs to be updated to 0.10.4

**Review by sideeffffect — CHANGES_REQUESTED** [review]
https://repo1.maven.org/maven2/ch/epfl/scala/scalafix-testkit_2.13.10/

File: project/plugins.sbt [hunk 3]
```diff
@@ -15,6 +15,6 @@ addSbtPlugin("org.scalameta"                     % "sbt-mdoc"
 addSbtPlugin("org.scalameta"                     % "sbt-scalafmt"                  % "2.4.6")
 addSbtPlugin("pl.project13.scala"                % "sbt-jcstress"                  % "0.2.0")
 addSbtPlugin("pl.project13.scala"                % "sbt-jmh"                       % "0.4.3")
-addSbtPlugin("ch.epfl.scala"                     % "sbt-scalafix"                  % "0.10.1")
+addSbtPlugin("ch.epfl.scala"                     % "sbt-scalafix"                  % "4")
```
Comment (sideeffffect) [inline project/plugins.sbt]: typo :smile_cat:
```suggestion
addSbtPlugin("ch.epfl.scala"                     % "sbt-scalafix"                  % "0.10.4")
```

**Comment by sideeffffect** [comment]
All tests green :tada: 
Better to use _Squash_ merge in this case :wink:

**Comment by sideeffffect** [comment]
@adamgfraser @mijicd ping :smile_cat:

**Review by mijicd — APPROVED** [review]

**Pull request merged**